File: SoftUni Lessons/Python Development/Python Fundamentals September 2019/Problems And Files/12 FUNCTIONS - Дата 9-ти октомври, 1830 - 2130/04. Orders.py
"""
Functions - Lab
Check your code: https://judge.softuni.bg/Contests/Practice/Index/1727#3

SUPyF2 Functions-Lab - 04. Orders

Problem:
Write a function that calculates the total price of an order and prints it on the console.
The function should receive one of the following products: coffee, coke, water, snacks; and a quantity of the product.
The prices for a single piece of each product are:
•	coffee - 1.50
•	water - 1.00
•	coke - 1.40
•	snacks - 2.00
Print the result formatted to the second decimal place.

Example:
Input:	    Output:

water
5	        5.00

coffee
2	        3.00
"""


def order():
    product = input()
    quantity = int(input())
    if product == "coffee":
        product = 1.50
    elif product == "coke":
        product = 1.40
    elif product == "water":
        product = 1.00
    elif product == "snacks":
        product = 2.00
    print(f"{(product * quantity):.2f}")


if __name__ == '__main__':
    order()

File: SoftUni Lessons/JavaScript Development/JS Fundamentals May 2023/10. Functions - Lab/04. Orders.js
function orders (product, count) {
    // MASK
    let prices = {
        'coffee': 1.50,
        'water': 1.00,
        'coke': 1.40,
        'snacks': 2.00
    };

    console.log(`${(prices[product] * count).toFixed(2)}`);
}

orders("water", 5 );
// 5.00

orders("coffee", 2);
// 3.00
